data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "work": "https://sinaimanuscripts/library.ucla.edu/work/1",
+  "associated_persons": [
+    {
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/42",
+      "attested_name_NS": "ܕܩܕܝܫܐ ܡܪܝ ܝܘܚܢܢ",
+      "attested_name": "Qadisho Mar Yuhanon",
+      "role": "attributed-author",
+      "source": "Final rubric on f. 152v"}
+  ],
```

Comment:
Here we run into the whole mess of problems of authorship and attribution. The work record will contain the information on the correct author (based on scholarly consensus, and noting any dissent/disputation/ambiguity). The manuscript may provide an attributed author, for whom we would want to create a person record and whose name form(s) we would want to capture. I'm not sure for SDP we need to worry about correct or incorrect attribution? We could instead rely on the authority file IDs, and if the one attested in the work ID is different from the one in the textual artifact record, we can infer it is a misattribution.

We could also glean from this data a work record that has all the persons to whom a record is misattributed included, as that might be interesting to a researcher. These, I think, could all be accomplished using linked data inferencing and querying.

Where the system of relying on mismatched author IDs (between attributed and scholar-asserted authorship) breaks down might be in cases of disputed authorship. However, even here I think we can reliably store that information in the work record rather than in the textual artifact record. This may need quite a bit more discussion.

Reply:
Using just attributed-author role in textual artifact and leaving the author info (including disputed, contested, etc.) in the work record is fine.

Reply:
Using just attributed-author role in textual artifact and leaving the author info (including disputed, contested, etc.) in the work record is fine.